.github/workflows/checkov.yaml

```diff
@@ -0,0 +1,20 @@
+name: "Checkov GitHub Action"
+on:
+  pull_request:
+    branches: [test, dev, qa, prod, main]
+
+jobs:
+  checkov:
+    name: checkov
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+          token: ${{ secrets.ENGINEERING_GITHUB_PERSONAL_ACCESS_TOKEN }}
+      - name: Run Checkov
+        id: checkov
+        uses: bridgecrewio/checkov-action@master
+        with:
+          download_external_modules: true
```

Comment:
This will not work well without a config file in repo root - pls see my active PR in terraform-aws-eks